### PR TITLE
Devx cypress examples

### PR DESCRIPTION
**Pull request by nadvolod**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -81,8 +81,15 @@ jobs:
             npm install
             npm run test.saucelabs.us
       - run:
-          name: "Run Nightwatch"
+          name: "Testrunner Toolkit Cypress examples"
```
Comment (wswebcreation) [inline .circleci/config.yml:84]: This fails, do we already want to run this in CI?

Comment (nadvolod) [reply .circleci/config.yml:84]: @wswebcreation thanks for being on top of this! I'm working on fixing it right now 👍

Comment (wswebcreation) [reply .circleci/config.yml:84]: Please remove SauceCTL from the pipeline, we haven't implemented it yet here in this CI (not as far as I know)

**Comment by wswebcreation** [comment]
By the way, you forgot to update the docs, will do that now

**Pull request merged**